src/integrations/generate-metadata.ts

```diff
@@ -0,0 +1,60 @@
+import type { AstroIntegration } from "astro";
+import fs from "fs";
+
+const rTitle = /@title: (.+)/;
+const rAuthor = /@author: (.+)/;
+const rTags = /@tags: (.+)/;
+const rImg = /@img: (.+)/;
+const rAddedOn = /@addedOn: (.+)/;
+
+const walk = () => {
+	const files = fs.readdirSync("./games/");
+	return files.filter((file) => file.endsWith(".js"));
+};
+
+const setup = () => {
+	const integration: AstroIntegration = {
+		name: "generate-metadata",
+		hooks: {},
+	};
+
+	integration.hooks["astro:config:done"] = () => {
+		const metadata: any = [];
+
+		walk().forEach((gameFile) => {
```

Comment:
suggest forcefully deleting any preexisting metadata.json file.  also it might make sense to add metadata.json to gitignore moving forward?

Reply:
I suppose removing metadata.json can be done as a separate pr, perhaps

Reply:
yea, the point of keeping it was for now was compatibility reasons